### DISCUSSION TITLE
Ödev'de Konu Paylaşımı

### DISCUSSION THREAD

**Discussion by caganseyrek** [gusanmaz/BMB311_OS_2023#7] [Q&A] [answered]
Hocam, grubumuzdaki herkesin içinde kod bulunan bir konu anlatması gerekir mi?
Sadece sözel konu anlatımı yapanları da kabul eder misiniz?

**Comment by gusanmaz** [comment, marked as answer]
Anlatilacak 9 farkli kod ornegi var. Gruplar en fazla 8 kisilik olabilecegi icin her grup uyesine anlatabilecegi bir konu atanabilir aslinda.

Kitaptaki konu anlatimlari ile kodlar buyuk olcude birbiri ile baglantili. Dolayisiyla her uyenin bir kodu ve o kitapta o kodla ilgili aciklamalarin yapildigi bolumu beraber anlatmasi daha dogru olur.

Koddan bagimsiz anlatilabilecek bazi konular var. Ornegin 26. pdf'de 26.2'ye kadar olan kisim koddan bagimsiz anlatilabilir. Grup uyelerinden biri boyle bir koddan bagimsiz bolumleri secip sadece oralari anlatirsa sorun olmaz.
